inc/wc-calypso-bridge-disable-publicize.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Disables publicize on API-created Products
+ *
+ * @since 0.2.3
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+add_action( 'publicize_save_meta', 'wc_calypso_bridge_maybe_disable_publicize', 10, 4 );
+
+/**
+ * Possibly Update post meta to disable publicize on Products
+ *
+ * @since 0.2.3
+ *
+ * @param bool $submit_post Should the post be publicized.
+ * @param int $post->ID Post ID.
+ * @param string $service_name Service name.
+ * @param array $connection Array of connection details.
+ */
+function wc_calypso_bridge_maybe_disable_publicize( $submit_post, $post_id, $service_name, $connection ) {
+    $trigger_strings = array( '/wp-json/wc/v', '/?rest_route=%2Fwc%2Fv' );
+    $should_disable_publicize = false;
```

Comment:
Minor: Maybe name this `is_rest_api_request` instead, since it's used in combination with the product type to decide if we should publicize or not.